_data/metadata_registry.yml

```diff
@@ -0,0 +1,28 @@
+# User Metadata
+- name: User Metadata
+  values:
+    - key: avatar
+      description: Avatar that graphical clients can show alongside the user's name
+      format: URL with an optional `{size}` substitution denoting the size to load in pixels
```

Comment:
Is it height or width?

Reply:
Original issue stated "square dimension" so both.